### PR TITLE
Fabric 7: Adding perf-test package.

### PR DESCRIPTION
**Pull request by dzearing**
This change introduces a `perf-test` package, representing a perf playground for testing scenarios and the impact it makes.

I expect more improvement here, but I'd like to get this started with anticipation of running performance tests in CI eventually. That way we can block changes that regress performance.

I've added it to Fabric 7 to start testing perf scenarios using react 16.8 with latest typescript.

As you can see it is quite simple now and could certainly be improved:

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8342)

### CODE REVIEW

File: apps/perf-test/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "perf-test",
+  "description": "Sample todo app using office-ui-fabric-react.",
```
Comment (KevinTCoughlin) [inline apps/perf-test/package.json]:
```suggestion
  "description": "Sandbox app for analyzing component performance.",
```

Or similar.. tho it doesn't really matter

Comment (ecraig12345) [reply apps/perf-test/package.json]: I'm also in favor of accurate comments, even in unpublished projects :)

File: apps/perf-test/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (ecraig12345) [inline apps/perf-test/index.html]: Do you need this file?

Comment (KevinTCoughlin) [reply apps/perf-test/index.html]: I think so, its where the App is rendered via `perf-test.js`. https://github.com/OfficeDev/office-ui-fabric-react/pull/8342/files#diff-f3e56f13bc159ea561c249d3a7cf897fR9

Comment (ecraig12345) [reply apps/perf-test/index.html]: Got it. In that case, maybe the title should be updated to something besides "Todo App" :)

File: apps/perf-test/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<html dir="ltr">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Todo App</title>
```
Comment (KevinTCoughlin) [inline apps/perf-test/index.html]:
```suggestion
    <title>Performance Sandbox</title>
```

**Pull request merged**